package.json

```diff
@@ -8,5 +8,8 @@
   },
   "keywords": [],
   "author": "",
-  "license": "ISC"
+  "license": "ISC",
+  "dependencies": {
+    "react-open-weather": "^1.1.3"
+  }
 }
```

Comment:
Could you undo this change? 
I don't think there is any need to add anything to this file.